core/src/main/java/feign/ReflectiveFeign.java

```diff
@@ -224,6 +224,27 @@ public RequestTemplate create(Object[] argv) {
       return template;
     }
 
+    private Object expandElements(Expander expander, Object value) {
```

Comment:
For consistency I think we should check instance of Iterable not, Collection or array. Currently, we don't have any special handling for arrays, while we do for Iterable (HeaderMap and QueryMap)